pkgs/tools/inputmethods/m17n-lib/default.nix

```diff
@@ -11,9 +11,11 @@ stdenv.mkDerivation rec {
   strictDeps = true;
 
   # reconf needed to sucesfully cross-compile
-  nativeBuildInputs = [ autoreconfHook pkg-config ];
-
-  buildInputs = [ m17n_db ];
+  nativeBuildInputs = [
+    autoreconfHook pkg-config
+    # requires m17n-db tool at build time
+    m17n_db
```

Comment:
The `m17n_db` package only contains architecture-independent files, even the `m17n-db` tool is a shell script.  So this shouldn't be an issue for cross-compilation.